Goal: 
1. Using public datasets to characterize the microbial composition of an environment and perform simple comparisons. 
2. Bulid my first database.
3. Become more comfortable with Git and GitHub.

Workflow:
1. Obtain 16S rRNA sequencing data from repositories.
2. Perform quality control using FastQC.
3. Process reads with DADA2 or QIIME2.
4. Assign taxonomy to the sequences using SILVA or Greengenes databases.
5. Visualize microbial diversity using Shannon Index and PCoA plots.

Portfolio Output:
Bar plots of relative microbial abundance.
Diversity metrics and principal coordinate analysis (PCoA).
